route_following_plugin/include/route_following_plugin.h

```diff
@@ -279,6 +280,9 @@ namespace route_following_plugin
         // Current vehicle pose in map
         geometry_msgs::PoseStamped pose_msg_;
         lanelet::BasicPoint2d current_loc_;
+        
+        // Currently executing maneuver plan from Arbitrator
+        cav_msgs::ManeuverPlanConstPtr current_maneuver_plan_;
```

Comment:
Can you do a quick find on current_maneuver_plan and make sure this doesn't cause confusion with other variables in this file?

Reply:
Yep, it's unique to this changeset.